installing/installing_bare_metal/upi/installing-bare-metal-network-customizations.adoc

```diff
@@ -0,0 +1,223 @@
+:boot-media!:
+:boot!:
+
+:boot-media: PXE environment
+:boot: pxe
+include::modules/installation-user-infra-machines-advanced-customizing-live.adoc[leveloffset=+3]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-serial-console.adoc[leveloffset=+4]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-ca-certs.adoc[leveloffset=+4]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-network-config.adoc[leveloffset=+4]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-iscsi-manual.adoc[leveloffset=+4]
+
+include::modules/installation-user-infra-machines-advanced-customizing-live-iscsi-ibft.adoc[leveloffset=+4]
+:boot-media!:
+:boot!:
+
+include::modules/installation-user-infra-machines-static-network.adoc[leveloffset=+3]
+
+include::modules/rhcos-enabling-multipath.adoc[leveloffset=+2]
+
+//iscsi using `coreos-installer install`
+include::modules/rhcos-install-iscsi-manual.adoc[leveloffset=+2]
+
+include::modules/rhcos-install-iscsi-ibft.adoc[leveloffset=+2]
+
+include::modules/installation-installing-bare-metal.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../../support/troubleshooting/troubleshooting-installations.adoc#monitoring-installation-progress_troubleshooting-installations[Monitoring installation progress] for more information about monitoring the installation logs and retrieving diagnostic data if installation issues arise.
+
+include::modules/cli-logging-in-kubeadmin.adoc[leveloffset=+1]
+
+include::modules/installation-approve-csrs.adoc[leveloffset=+1]
+
+include::modules/installation-operators-config.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../../support/troubleshooting/troubleshooting-installations.adoc#installation-bootstrap-gather_troubleshooting-installations[Gathering logs from a failed installation] for details about gathering data in the event of a failed {product-title} installation.
+* See xref:../../../support/troubleshooting/troubleshooting-operator-issues.adoc#troubleshooting-operator-issues[Troubleshooting Operator issues] for steps to check Operator pod health across the cluster and gather Operator logs for diagnosis.
+
+include::modules/registry-removed.adoc[leveloffset=+2]
+
+include::modules/installation-registry-storage-config.adoc[leveloffset=+2]
+
+include::modules/installation-registry-storage-block-recreate-rollout-bare-metal.adoc[leveloffset=+2]
+
+include::modules/installation-complete-user-infra.adoc[leveloffset=+1]
+
+include::modules/cluster-telemetry.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring] for more information about the Telemetry service
+
+== Next steps
```

Comment:
No need for periods after these ref. Perhaps fodder for a follow-up.

Reply:
Follow up, yes. From what I can see, the UPI docs need a full Vale linting at the least.